packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -36,6 +34,10 @@ const retryTimeSpanInMillisecond = 3000;
 export class TeamsUserCredential implements TokenCredential {
   private readonly config: AuthenticationConfiguration;
   private ssoToken: AccessToken | null;
+  private initialized: boolean;
+  private msalInstance?: PublicClientApplication;
+  private tid?: string;
+  private loginHint?: string;
```

Comment:
Can we remove all these private properties, and in the future we do not need to update this part?